src/ifp/README.md

```diff
@@ -35,45 +51,71 @@ die =  ( 0, 0 )
 
 ### Make Tracks
 
-The `initialize_floorplan` command removes existing tracks. Use the
-`make_tracks` command to add routing tracks to a floorplan.
+The `initialize_floorplan` command removes existing tracks. 
 
-```
-make_tracks [layer]
-            [-x_pitch x_pitch]
-            [-y_pitch y_pitch]
-            [-x_offset x_offset]
-            [-y_offset y_offset]
+Use the `make_tracks` command to add routing tracks to a floorplan.
+
+```tcl
+make_tracks 
+    [layer]
+    [-x_pitch x_pitch]
+    [-y_pitch y_pitch]
+    [-x_offset x_offset]
+    [-y_offset y_offset]
 ```
 
-With no arguments `make_tracks` adds X and Y tracks for each routing layer.
-With a `-layer` argument `make_tracks` adds X and Y tracks for layer with
-options to override the LEF technology X and Y pitch and offset.
+#### Options
 
-### Place pins around core boundary
+| Switch Name | Description |
+| ----- | ----- |
+| `layer` | select layer name to make tracks for (default all layers). |
```

Comment:
```suggestion
| `-layer` | select layer name to make tracks for (default all layers). |
```

Reply:
Layer is not a keyword arg so I didn't add a `-` in front of it

Reply:
ok, got it. you can resolve this.